datumaro/plugins/data_formats/segment_anything/exporter.py

```diff
@@ -0,0 +1,173 @@
+# Copyright (C) 2023 Intel Corporation
+#
+# SPDX-License-Identifier: MIT
+
+
+import logging as log
+import os
+import os.path as osp
+from itertools import chain
+from typing import List, Union
+
+from pycocotools import mask as mask_utils
+
+from datumaro.components.annotation import AnnotationType, Ellipse, Polygon
+from datumaro.components.errors import DatumaroError, MediaTypeError
+from datumaro.components.exporter import Exporter
+from datumaro.components.media import Image
+from datumaro.util import NOTSET
+from datumaro.util import annotation_util as anno_tools
+from datumaro.util import dump_json_file, mask_tools
+
+
+def replace(json_data, key, value_new):
```

Comment:
when is this used?

Reply:
Thanks for catching it. It was a redundant function after review with Vinnam. Removed in https://github.com/openvinotoolkit/datumaro/pull/1005/commits/8c6056e655d9f0e630c1d56f19530cb99c713882